### ISSUE TITLE
medgen umls

### ISSUE THREAD

**Issue by matentzn** [monarch-initiative/mondo#7799]

**Comment by kanems** [comment]
FYI a few of these _are_ in scope for MedGen, but they have CN-style CUIs. We found a bug in our script that generates the IDMappings report that was excluding a subset of our CN-CUI linked terms and it's fixed now so that those are in the report.  I can try to pull the CUIs from your table above, or if you download a more recent IDMappings file it should fill in a decent chunk of the human diseases above, it won't be a UMLS-sourced CUI (because UMLS doesn't have CUIs specific for these terms) but it'll be something.

**Comment by matentzn** [comment]
@kanems thanks a ton! We will keep syncing with your IDMappings file!

We decided not to sync CN-CUIs #7683 anymore, just the Mondo->Medgen UIDs and Mondo -> UMLS mappings from Medgen! Thank you very much for checking in!

**Comment by kanems** [comment]
@matentzn Just a heads up that I noticed a few Mondo IDs seem to be out of sync with CUI xrefs relative to current MedGen mappings.
Example: http://purl.obolibrary.org/obo/MONDO_0024548 reports [UMLS:C1849193](https://uts.nlm.nih.gov/uts/umls/concept/C1849193) as the CUI mapping But we moved this to the parent record after the 2025AB UMLS release:
Relevant row from the IDmappings report:
` C1849193|Peeling skin syndrome|MONDO:0019347|MONDO|  `

Looks like there are ~12 Mondo ID:CUI mismatches from the March '26 Mondo release. Do you all want a report of those or is it just easier to check on the pipeline to refresh the CUI mappings from MedGen?

**Comment by matentzn** [comment]
Hey @kanems 

I noticed a bug recently in the mondo-ingest which precluded the medgen ingest to be updated correctly. This is what is in the development snapshot now *after fixing the issue*. Is this correct?

```
[Term]
id: MONDO:0024548
name: peeling skin syndrome 1
xref: UMLS:C5679693 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:1885521"}
```

**Comment by kanems** [comment]
@matentzn Yes, that's right. I'll check our internal QC query again after we process the April release, let you know if I see any lingering gaps. Thanks!

**Comment by matentzn** [comment]
Thank you @kanems!

Note that this change will be in the Mondo release in 10 days from now!

I will also close this issue because, we have, indeed deferred our UMLS mapping efforts to MedGen!
